tests/Builder/Stage/CurrentOpStageTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Tests\Builder\Stage;
+
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Query;
+use MongoDB\Builder\Stage;
+use MongoDB\Tests\Builder\PipelineTestCase;
+
+/**
+ * Test $currentOp stage
+ */
+class CurrentOpStageTest extends PipelineTestCase
+{
+    public function testInactiveSessions(): void
+    {
+        $pipeline = new Pipeline(
+            Stage::currentOp(
+                allUsers: true,
+                idleSessions: true,
+            ),
+            Stage::match(
+                active: false,
+                transaction: Query::exists(true),
```

Comment:
We could have `true` as default value. So that it can be written: `Query::exists(),`

Reply:
Sounds like a good idea 👍

Reply:
[PHPLIB-1384](https://jira.mongodb.org/browse/PHPLIB-1384)